package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "contract-addresses",
+  "version": "0.0.1",
+  "description": "A place to store all the contract addresses for the Zero ecosystem",
+  "main": "index.ts",
+  "scripts": {
+    "build": "tsc"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zer0-os/contract-addresses.git"
```

Comment:
Can we rename this GitHub repo to `zer0-contracts` please, and update these references.